jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -367,6 +367,12 @@ def step900_collect_driver_artifacts() {
     // NOTE: this DSL step requires the puppet-gatling-jenkins plugin.  It also
     // depends on some data that gets created via 025_collect_facter_data.sh
     puppetGatlingArchive()
+
+    // Always archive the simulation.log.gz from the run.
+    archive "simulation-runner/results/**/*.log.gz"
+    dir "simulation-runner/results" {
+        deleteDir
```

Comment:
Is this really necessary if the box will just be razored later?

Reply:
this is on the jenkins machine.

Reply:
oh, then I have no idea what archive does.

Does this still allow the gatling reports to be displayed in jenkins?

Reply:
i stuck it in `step900_collect_driver_artifacts` as that seemed like the right place for this.

Reply:
also related question: if puppetGatlingArchive() doesn't do this, what does it do

Reply:
I believe the reports are served from the `jobs/$job/builds/$build_number` directory, and this operates in the `jobs/$job/workspace` area. I'll know for sure after the current run completes.

Reply:
puppetGatlingArchive seems to be the magic to inline the html report. the important bits seem to be here: https://github.com/puppetlabs/puppet-gatling-jenkins-plugin/blob/be8328bc9d39c638cf0a123beb8a3ec5bfe98c2e/src/main/java/com/puppetlabs/jenkins/plugins/puppetgatling/PuppetGatlingPublisher.java

Reply:
Ah, okay. If you don't mind I'd like to wait for that to be sure, don't want to delete this and have it break for a first time run of anything

Reply:
Waiting is fine. I'm pretty confident based on poking around in the jenkins FS.